llvm/test/tools/sycl-post-link/composite-spec-constant.ll

```diff
@@ -23,7 +23,17 @@
 ; CHECK: %[[#POD:]] = call %struct._ZTS3POD.POD @"_Z29__spirv_SpecConstantCompositeAstruct._ZTS1A.Aclass._ZTSN2cl4sycl3vecIiLi2EEE.cl::sycl::vec"([2 x %struct._ZTS1A.A] %[[#NA]], %"class._ZTSN2cl4sycl3vecIiLi2EEE.cl::sycl::vec" %[[#B]]), !SYCL_SPEC_CONST_SYM_ID ![[#MD:]]
 ; CHECK: store %struct._ZTS3POD.POD %[[#POD]]
 ;
+; Test checks related to 2020 API for composite specialization constants.
```

Comment:
this does not seem to have any 2020 specifics, am I missing something?

Reply:
These checks are just related to function with new intrinsic API calls, but you're right that IR doesn't look different.
I'll remove this mark as a bit confusing one.